script/makeinstaller.sh

```diff
@@ -43,14 +43,23 @@ do
     touch "${build_dir_plugins}/$i.py"
 done
 
+echo "Creating images"
+# Create all of our various images
+for size in 16 32 48 256; do
+  inkscape --no-gui --export-png="icon-${size}.png" -w ${size} -h ${size} icon.svg
```

Comment:
This will require `inkscape` to be install on the TravisCI machine doing the build... :(

Reply:
I would actually replace the whole loop with 

```
convert -density 256x256 -background transparent icon.svg -define icon:auto-resize -colors 256 icon.ico
```

ImageMagick can do it all. Using ImageMagick 7.0.8, the ico looks decent and you get all the sizes generated for you. The `optipng` stage does nothing in the end as the `ico` file ends up the same size either way :)

Reply:
My main issue with imagemagick is that it doesn't (unless they fixed it recently) do a very good job with svg -> png conversion. We still want the `optipng` stage as other teams (https://github.com/streamlink/streamlink-chocolatey/issues/19) either might need it. I've pushed a new commit, let me know what you think if travis passes on it.